### PR TITLE
update the stucyure and add functions

### PR DESCRIPTION
**Pull request by alfred-malope**

### CODE REVIEW

**Review by Penelope32 — APPROVED** [review]

File: BursaryManagementAPI/Controllers/BBDAdminController.cs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        }
+
+        //[Authorize(Roles = Roles.BBDAdmin)]
+        [HttpPost("{applicationId}/approve")]
+        public ActionResult ApproveApplication(int applicationId)
+        {
+            try
+            {
+                _BBDAdminBLL.ApproveApplication(applicationId);
+                return Ok("Application approved successfully!");
```
Comment (ndzalamabasa) [inline BursaryManagementAPI/Controllers/BBDAdminController.cs:66]: With this response, are able to get it as an object so we can access the value? I've been struggling with this one earlier.

**Review by ndzalamabasa — COMMENTED** [review]
Looks good @alfred-malope, just a 1 question below

File: BursaryManagementAPI/Controllers/BBDAdminController.cs (hunk 1 above)
Comment (alfred-malope) [reply BursaryManagementAPI/Controllers/BBDAdminController.cs:66]: image


Soething like this?

Comment (ndzalamabasa) [reply BursaryManagementAPI/Controllers/BBDAdminController.cs:66]: Not sure how to achieve it on dotnet, I'll check also on my side

**Review by ndzalamabasa — APPROVED** [review]

**Pull request merged**